content/browser/renderer_host/render_widget_host_view_child_frame.cc

```diff
@@ -135,6 +135,13 @@ void RenderWidgetHostViewChildFrame::SetFrameConnector(
       if (input_transfer_handler) {
         host()->RemoveInputEventObserver(input_transfer_handler);
       }
+      auto* manager = root_view->GetTouchSelectionControllerClientManager();
+      if (manager) {
+        // We have managers in Aura and Android, as well as outside of content/.
+        // There is no manager for Mac OS.
+        selection_controller_client_.reset();
+        manager->RemoveObserver(this);
```

Comment:
Do we need this change? Is it already covered by DetachFromTouchSelectionClientManagerIfNecessary() above  in line 129?